.circleci/config.yml

```diff
@@ -10,10 +10,6 @@ jobs:
       - image: circleci/node:10.14.0-browsers
     steps:
       - checkout
-      - restore_cache:
-          keys:
-          - v2-dependencies-{{ checksum "package-lock.json" }}
-          - v2-dependencies-
```

Comment:
Are we no longer caching in circle ci? We can probably remove the save_cache portion below as well if so

Reply:
Yes, I am planning to remove that because of circle ci build issue.